cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -1209,8 +1240,8 @@ Rolling up the above using ``rollup=workflow`` would group the programs together
 
 .. _http-restful-api-metadata-run:
 
-Retrieving Metadata for a Program Run
-=====================================
+Retrieving Metadata for a Program Run (Deprecated)
```

Comment:
retrieving metadata for a program run is deprecated, or just the part that it is aggregated?

Reply:
Only the behavior where the behavior which is mentioned here. After 5.0 we will remove this whole section.

Reply:
Why will this whole section be removed? Because metadata for a program run won't be useful?

Reply:
Kinda, https://issues.cask.co/browse/CDAP-13721

Reply:
More context: https://github.com/caskdata/cdap/pull/10341#issue-201651627